src/구명보트.js

```diff
@@ -0,0 +1,25 @@
+function solution(people, limit) {
+  let result = 0;
+  let left = 0;
+  let right = people.length - 1;
+
+  people.sort((a, b) => b - a);
+
+  while(left < right) {
+    let sum = people[left] + people[right];
+    result++;
+
+    if(sum > limit){
+      left++;
+    } else {
+      left++;
+      right--;
+    }
+  }
+
+  if(left === right) {
+    result++;
+  }
```

Comment:
```suggestion
  while(left <= right) {
    let sum = people[left] + people[right];
    result++;

    if(sum > limit){
      left++;
    } else {
      left++;
      right--;
    }
  }
```

안녕하세요 다은님. 제가 엣지케이스를 많이 생각해보진 않았는데 제 경우에는 if문을 따로 나누지 않아도 해결되었습니다. 이렇게 하는게 좀 더 가독성이 좋지 않을까 싶습니다. 수고하셨습니다!

Reply:
while문 조건을 바꾸면 굳이 안 나누어도 되겠군요! 감사합니다!